addons/udes_stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -351,7 +325,6 @@ def add_extra_pickings(self, picking_type_id, limit=1):
             raise ValidationError(_("No more work to do."))
 
         picking_type = pickings.picking_type_id
-        picking_type.ensure_one()
```

Comment:
What not needed ensure_one(), i think still by code can be more than one picking_type

Reply:
I think you are right, it was like that on udes11, ported in same way. But it doesnt seem a possibility that can be more than one picking, maybe just extra precautional was the reason of adding it first in udes11.